src/lib.rs

```diff
@@ -417,7 +413,7 @@ impl Decoder {
             } else {
                 let inner = InnerPicture { pic };
                 Ok(Picture {
-                    inner: Arc::new(inner),
+                    inner: Rc::new(inner),
```

Comment:
Uh why? Now `Picture`s are not `Send` anymore, but they should be or not?

Reply:
Hmm clippy raised this issue, but indeed, it might need to be ignored, I'll check.

Reply:
`Send+Sync` is not implemented for `InnerPicture`... that's why clippy complained. I can add those.

fwiw the gst dav1ddec plugin still works if the inner picture is in a `Rc`.

Reply:
Something to look at separately I guess, unless it's absolutely clear that `Picture` is indeed thread-safe. It's immutable, isn't it?

Reply:
Yes, afaik it is immutable.

Well, for now I'll restore the previous code, add a clippy override and open a separate issue.

Reply:
https://github.com/rust-av/dav1d-rs/issues/95